plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java

```diff
@@ -186,8 +186,7 @@ private void resolveFromChannels(MavenArtifact artifact) throws UnresolvedMavenA
 
     public void done(Path home) throws MavenUniverseException, IOException {
         ChannelManifest channelManifest = channelSession.getRecordedChannel();
-        final ManifestVersionRecord currentVersions = new ManifestVersionResolver(localCachePath, system)
-                .getCurrentVersions(channels);
+        final ManifestVersionRecord currentVersions = ManifestVersionResolver.getCurrentVersions(channelSession);
```

Comment:
Is there a reason for this change?

Reply:
@jamezp I deprecated the old method - it was relying on a hack I want to get rid of. The new method uses changed wildfly-channel API to achieve the same goal more cleanly.

Reply:
Yeah, that was really @spyrkob 's request.